webapp/src/main/webapp/WEB-INF/conf/news/application-registry-configuration.xml

```diff
@@ -76,6 +76,32 @@
                     </field>
                   </object>
                 </value>
+                <value>
```

Comment:
this configuration should not be added here in news codebase but into layout-management codebase, the same location of the portlet itself. This PR should be closed